### DISCUSSION TITLE
First alert Zwave Plus combo detector not getting added

### DISCUSSION THREAD

**Discussion by cvkattookaran** [zwave-js/zwave-js#7430] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
I keep getting the following error during the interview process
2024-11-21T21:36:54.213Z CNTRLR   [Node 124] Error during node interview: Argument validation failed:
                                  Expected parameter nodeIds to be an Array<number>, got 1 (ZW0322)

What did you expect to happen instead?
Smoke detector to get added

  1. Try adding FirstAlert combo smoke detector and will not add it

Anything else we should know?

### Software versions

zwave-js-ui: 9.27.4.40bcf6b
zwave-js: 14.3.4

HA 
Core
2024.11.1
Frontend
20241106.2

HA and Zwave JS are on docker

### Z-Wave Controller (Stick/Dongle/...)

Zooz 800 Controller

### Device information

Manufacturer: FirstAlert
Model name: ZCOMBO
Node ID: 125/124

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2024-11-21.log](https://github.com/user-attachments/files/17853349/zwavejs_2024-11-21.log)
[zwavejs_current.log](https://github.com/user-attachments/files/17853350/zwavejs_current.log)

**Comment by AlCalzone** [comment]
Escalated to an issue: #7434

**Reply by AlCalzone** [reply]
And fixed in https://github.com/zwave-js/node-zwave-js/releases/tag/v14.3.5

**Comment by cvkattookaran** [comment]
Same error during re-interview after upgrading.
2024-11-22T14:29:37.695Z CNTRLR   [Node 125] Error during node interview: Argument validation failed:
                                  Expected parameter nodeIds to be an Array<number>, got 1 (ZW0322)
zwave-js-ui: 9.27.5.b68aa39
zwave-js: 14.3.4
home id: 3630087349
home hex: 0xd85ebcb5

**Reply by AlCalzone** [reply]
> zwave-js: 14.3.4

That's not the latest driver version yet. Z-Wave JS UI still needs to be updated. Let me do that right now.

**Reply by AlCalzone** [reply]
9.27.6 has just been released and should be available as a Docker image soon.

**Reply by cvkattookaran** [reply]
another one now.
zwave-js-ui: 9.27.6.f350ca8
zwave-js: 14.3.5
home id: 3630087349
home hex: 0xd85ebcb5
2024-11-22T15:16:18.316Z CNTRLR   [Node 125] Error during node interview: Argument validation failed:
                                  Expected parameter allowCache to be a boolean, got undefined (ZW0322)

**Reply by AlCalzone** [reply]
Fix coming up. Will let you know once it's released.

**Reply by AlCalzone** [reply, marked as answer]
Fixed in the upcoming release 14.3.6 / Z-UI 9.27.7. I'm working on releasing both right now.

**Reply by AlCalzone** [reply]
Released, Docker container is building

**Reply by cvkattookaran** [reply]
thank you and it works.
